packages/dds/tree/src/simple-tree/index.ts

```diff
@@ -198,7 +199,12 @@ export type {
 	ArrayNodeCustomizableSchema,
 	ArrayNodePojoEmulationSchema,
 } from "./arrayNodeTypes.js";
-export type { ArrayNodeSchema, isArrayNodeSchema } from "./arrayNodeTypes.js";
+export { ArrayNodeSchema, isArrayNodeSchema } from "./arrayNodeTypes.js";
```

Comment:
removal of `type` here fixes a bug in the last change which prevented use of narrowing APIs.